ldes-fragmentisers/ldes-fragmentisers-geospatial/src/main/java/be/vlaanderen/informatievlaanderen/ldes/fragmentisers/geospatial/GeospatialFragmentNamingStrategy.java

```diff
@@ -0,0 +1,21 @@
+package be.vlaanderen.informatievlaanderen.ldes.fragmentisers.geospatial;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.config.LdesFragmentNamingStrategy;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.ldesfragmentrequest.entities.FragmentPair;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static be.vlaanderen.informatievlaanderen.ldes.fragmentisers.geospatial.constants.GeospatialConstants.FRAGMENT_KEY_TILE;
+
+public class GeospatialFragmentNamingStrategy implements LdesFragmentNamingStrategy {
+	
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
format should come from configuration

Reply:
https://github.com/Informatievlaanderen/VSDS-LDESServer4J/issues/160